src/pulse3D/exceptions.py

```diff
@@ -19,42 +19,34 @@ class PeakDetectionError(Exception):
     pass
 
 
+class DuplicateWellsFoundError(Exception):
+    pass
+
+
+class IncorrectOpticalFileFormatError(Exception):
+    pass
+
+
 class TwoFeaturesInARowError(PeakDetectionError):
     """There must always be a peak in between valleys and vice-versa."""
 
-    def __init__(
```

Comment:
I think this is a formatting issue. Unless this is older code and its reformatting for the updated format.

Reply:
My formatter removes the magic comma and formats to the line length of 110